### PR TITLE
using bem methodologies for classname

### PR DESCRIPTION
**Pull request by GribinetDaniel**

### CODE REVIEW

File: packages/user-interface/src/components/ErrorPage/ErrorPage.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './index.scss';
+export function ErrorPage() {
+  return (
+    <div className='error-page'>
+      <img
+        className='error-page__image'
+        src={require('./images/error-image.jpg')}
```
Comment (H7GhosT) [inline packages/user-interface/src/components/ErrorPage/ErrorPage.tsx]: Use import not require
`import errorImgSrc from "....jpg"`

File: packages/user-interface/src/App.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { useContext } from 'react';
-import Login from './pages/Login';
-import Register from './pages/Register';
-import Homepage from './pages/HomePage';
+import { Login } from './pages/Login';
+import { Register } from './pages/Register/Register';
+import Homepage from './pages/Homepage';
```
Comment (H7GhosT) [inline packages/user-interface/src/App.tsx]: Be consistent with default and named exports

File: packages/user-interface/src/components/SignUp.tsx [hunk 3]
```diff
@@ -1,48 +1,64 @@
...
+      </label>
       <input
         type='text'
         id='username'
         name='username'
         placeholder='Enter your username'
+        className='register__input'
         value={username}
         onChange={handleInput}
       />
```
Comment (H7GhosT) [inline packages/user-interface/src/components/SignUp.tsx]: Create component for input, so you don't have to duplicate your styles

**Review by H7GhosT — CHANGES_REQUESTED** [review]

File: packages/user-interface/src/components/ErrorPage/ErrorPage.tsx (hunk 1 above)
Comment (GribinetDaniel) [reply packages/user-interface/src/components/ErrorPage/ErrorPage.tsx]: La mine nu merge cu import, din cauza asta si i-am data require

File: packages/user-interface/src/types/inputProps.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export interface InputProps {
```
Comment (H7GhosT) [inline packages/user-interface/src/types/inputProps.tsx]: Place the InputProps right in the file of the component

**Review by H7GhosT — CHANGES_REQUESTED** [review]

**Pull request merged**